app-network/doggo/autobuild/prepare

```diff
@@ -0,0 +1,2 @@
+GO_LDFLAGS=("-X" "main.buildVersion=$PKGVER")
```

Comment:
Move these to `defines`, you were instructed to move them here because you called the `date` command and command outputs are not allowed in `defines`.